numpy/distutils/misc_util.py

```diff
@@ -2326,6 +2326,41 @@ def get_info(name):
                 return g.get(name, g.get(name + "_info", {}))
 
             def show():
+                """
+                Print information about various resources (libraries, library
+                directories, include directories, etc.) in the system on which
+                NumPy was built.
+
+                See Also
+                --------
+                get_include : Returns the directory containing NumPy C
+                              header files.
+
+                Notes
+                -----
+                Classes specifying the information to be printed are defined
+                in the ``numpy.distutils.system_info`` module.
```

Comment:
```suggestion
                in the `numpy.distutils.system_info` module.
```
Should produce a link here